src/integrationtest/python/doc_traversals_tests.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from janusgraph_container import JanusGraphContainer
+from janusgraph_python.driver.janusgraph_remote_connection_builder import JanusGraphRemoteConnectionBuilder
+from gremlin_python.structure.graph import Graph
+
+
+class TestDocTraversals(unittest.TestCase):
```

Comment:
Are you actually using the tested traversals here any where in the docs? I at least could find them with a quick search.

Reply:
Makes sense. I added this to replicate a generic behaviour in one on JanusGraph sub repos. Strange I can't find it anywhere.
I'm open to removing this though since its not being used and essentially the functionality this test is supposed to do is already tested as part of other integration tests.
Going to remove in next commit.